Assignment1-JavaScript/palindromePermutation.js

```diff
@@ -0,0 +1,25 @@
+function palindromePermutation(str){
+    let charCount = new Set();
+    str = str.toLowerCase().replace(/[^a-z]/g, '');
```

Comment:
This is a cool RegEx but becareful because it only allows the letters a-z. This should work for all characters that aren't whitespace. To remove just whitespace, consider the RegEx /\s/g